### ISSUE TITLE
EM: Tiled service publishing as hosted service is failing

### ISSUE THREAD

**Issue by jessnoo** [Esri/ops-server-config#1083]
@conklinbd I did one last test for hosted tile service publishing on the nsec portal and it failed also... 

![image](https://cloud.githubusercontent.com/assets/6181441/10202568/60c73c64-6766-11e5-9a29-9eab84487863.png)

map doc and data: \\mrhyde\public\Jess\OPSServer

**Comment by conklinbd** [comment]
@jessnoo I tried publishing the same tiled service and succeeded. 
after copying your zip file I changed 3 things.
- The data source was broken so I reset the data source and change the map to relative paths (probably not the cause)
-I change the feature class name to all lower case (possibly the cause)
-I changed the field name for Evacuation Zones (the symbology field) to all lower case (probably the cause).

After that I published to the server. I didn't know what scales to caches so I only cached a couple of levels  but it looks like it works.
https://nsec.esri.com/arcgis/home/item.html?id=8faa8be621c44fe1b7a70a822469fad4

Let me know if you want to try these steps yourself, or if you just want me to reassign the item I created back to you.
